allensdk/brain_observatory/behavior/data_objects/metadata/subject_metadata/reporter_line.py

```diff
@@ -68,13 +68,12 @@ def parse(reporter_line: Union[Optional[List[str]], str],
         if reporter_line is None:
             if warn:
                 warnings.warn('Error parsing reporter line. It is null.')
-            return None
+            return str(None)
```

Comment:
I assume we're going to have the same problem with the `driver_line.py` data object in the same `subject_metadata/` directory. That will need to be changed as well.

Also: would you mind, instead of casting the returns of this function into strings, handle the casting in `__init__` as in this PR

https://github.com/AllenInstitute/AllenSDK/pull/2318/files

I don't love the fact that we have to cast `None` to `'None'`. If we change it in `__init__` then, if we ever find a solution that lets us keep the result as a nonetype, we only have to change the single line of code in `__init__.py`, rather than changing all the return statements in this method.

Reply:
move conversion to subject_metadata.to_nwb